lib/generators/init-generator.js

```diff
@@ -378,15 +374,29 @@ module.exports = class InitGenerator extends Generator {
 						);
 					}
 				}
-			})
-			.then(() => {
-				if (this.configuration.config.webpackOptions.entry.length === 0) {
-					oneOrMoreEntries.forEach(prop => {
-						this.configuration.config.webpackOptions.plugins.push(
-							createCommonsChunkPlugin(prop)
-						);
-					});
-				}
+				// add splitChunks options for transparency
+				// defaults coming from: https://webpack.js.org/plugins/split-chunks-plugin/#optimization-splitchunks
+				this.configuration.config.topScope.push(tooltip.splitChunks());
+				this.configuration.config.webpackOptions.optimization = {
+					splitChunks: {
+						chunks: "'async'",
+						minSize: 30000,
+						minChunks: 1,
+						// for production name is recommended to be off
+						name: !this.isProd,
+						cacheGroups: {
+							vendors: {
+								test: "/[\\\\/]node_modules[\\\\/]/",
+								priority: -10
+							},
+						},
+						default: {
```

Comment:
This didn't work when trying to build a web app using this PR. Removing `default` makes the app run. I think maybe that you can use the object entries from when the user types what the names of their entries should be to add chunkGroups. Say if I answer yes to the first question at the CLI and type that I've got 2 entries, `app` and `utils`, then you should use that when setting chunk groups in addition to the node_modules for vendor. Makes sense?